### PR TITLE
Fix unicode comparision and counting error for test262

### PR DESCRIPTION
**Pull request by yuit**

### CODE REVIEW

File: src/harness/harness.ts [hunk 1]
```diff
@@ -1642,7 +1647,8 @@ module Harness {
         }
 
         function writeComparison(expected: string, actual: string, relativeFilename: string, actualFilename: string, descriptionForDescribe: string) {
-            if (expected != actual) {
+            var encoded_actual = (new Buffer(actual)).toString('utf8')
```
Comment (mhegazy) [inline src/harness/harness.ts]: should not we check if buffer is available or that we are running on node before using this?

Comment (DanielRosenwasser) [inline src/harness/harness.ts]: Browserify provides it, but it uses typed arrays. You might need to `require("buffer")`.

File: tests/webhost/webtsc.ts [hunk 2]
```diff
@@ -4,8 +4,12 @@ module TypeScript.WebTsc {
 
     declare var RealActiveXObject: { new (s: string): any };
 
-    function getWScriptSystem(): System {
+    function getWScriptSystem() {
         var fso = new RealActiveXObject("Scripting.FileSystemObject");
```
Comment (DanielRosenwasser) [inline tests/webhost/webtsc.ts]: `RealActiveXObject`??

File: src/harness/harness.ts [hunk 3]
```diff
@@ -1298,8 +1298,13 @@ module Harness {
                 return diagnostic.filename && isLibraryFile(diagnostic.filename);
             });
 
+            var test262HarnessDiagnostics = ts.countWhere(diagnostics, diagnostic => {
```
Comment (DanielRosenwasser) [inline src/harness/harness.ts]: `numTest262HarnessDiagnostics` - this is the count, not the diagnostics.

File: src/harness/harness.ts (hunk 1 above)
Comment (yuit) [inline src/harness/harness.ts]: This Buffer is the native node buffer. Though browserify version will be nicer as we can run in browser too. There will be additional installation to use it so I will let everyone know once checkin

Comment (DanielRosenwasser) [inline src/harness/harness.ts]: We already use browserify, it's one of our dev dependencies in Node, so might as well try to make it work there as well.

**Pull request merged**